documentation/getting-started/README.md

```diff
@@ -5,44 +5,51 @@ permalink: /documentation/getting-started/
 
-![The Explorer panel](../../images/getting-started/explorer-panel.png)
+![The Welcome page](../../images/getting-started/welcome-page.png)
 
-Next, create a test fixture for the sample webpage tests.
-In the **Explorer** panel, click the **Create a New Fixture** button, and in the invoked **New Fixture** dialog, specify the fixture name and the tested page's URL. Click **Create**.
+You can also organize your tests and start the recording from the [Explorer](../user-interface/explorer-panel.md) panel. Follow the steps:
 
-![The New Fixture dialog](../../images/getting-started/new-fixture-dialog.png)
+1. First, create a [test directory](../guides/organize-tests.md#test-directory) to store your tests. Then go to TestCafe Studio, open the **File** menu, click **Open Tests Directory** and navigate to the created directory. You can view the directory's contents in the **Explorer** panel.
 
-The new fixture is added to the test directory.
+2. Create a [fixture](../guides/record-tests/README.md#create-fixtures) for the sample webpage tests. To do this, click the ![Create a New Fixture button](../../images/getting-started/create-new-fixture-icon.svg) **Create a New Fixture** button in the **Explorer** panel. This invokes the **New Fixture** dialog where you can specify the fixture name and the tested page's URL ([http://devexpress.github.io/testcafe/example](http://devexpress.github.io/testcafe/example)).
 
-To start recording a test, click the **Record a New Test** button in the **Explorer** panel.
-TestCafe Studio launches the default browser with the tested site and starts recording your activity on the webpage.
+    ![The New Fixture dialog](../../images/getting-started/new-fixture-dialog.png)
 
-Perform the following actions on the webpage:
+    Click **Create**. The new fixture is added to the test directory.
 
-* Enter a name in the **Your name** field.
-* Check the **Support for testing on remote devices** and **Advanced traffic and markup analysis** checkboxes.
-* Click the **Submit** button.
+3. Select a browser where you want to test the webpage in the [Record Configurations](../guides/record-tests/README.md#record-configurations) drop-down menu on the toolbar.
+
+    ![Record configuration](../../images/getting-started/record-configuration.png)
+
+4. Start recording a test. To do this, click the ![Record button](../../images/getting-started/record-test-icon.png) **Record a New Test** button in the **Explorer** panel. TestCafe Studio launches the browser with the tested site and starts recording your activity on the webpage.
+
+## Interact with the Tested Webpage
+
+Perform the following actions on the tested webpage opened in the browser:
```

Comment:
Perform the following actions on the tested webpage opened in the browser: -> Perform the following actions on the tested webpage: